properties/base.py

```diff
@@ -155,10 +155,15 @@ def __init__(self, **kwargs):
                                'or attribute'.format(key))
             setattr(self, key, kwargs[key])
 
-    def _get(self, name):
+    def _get(self, name, default=None):
         if name in self._backend:
             return self._backend[name]
 
+        if default is utils.undefined:
```

Comment:
Just to tidy up loose ends, this was fixed here: https://github.com/3ptscience/properties/commit/7d1e4d662d50d55303e908ba5e30316d9a338862